test/test_ops.py

```diff
@@ -719,6 +723,103 @@ def _case_five_transform(t):
             with self.assertRaises(RuntimeError, msg=msg_fail):
                 op_out(out=out)
 
+    # Tests that
+    # 1. The operator's output for physically conjugated tensors and conjugate view tensors
+    # produces the same value
+    # 2. The gradients are same in both cases mentioned in (1)
+    # 3. If the operator's inplace variant is supported, tests that the inplace operation
+    #    produces the correct value when called on a conjugate view tensor and that the output
+    #    has its conj bit set to true
+    # This test only runs for C -> R and C -> C functions
+    # TODO: add tests for `R->C` functions
+    # Note: This test runs for functions that take both tensors and tensorlists as input.
+    @ops(op_db, allowed_dtypes=(torch.cfloat,))
+    def test_conj_view(self, device, dtype, op):
```

Comment:
This is a cool test and it looks like it has a lot of attention to detail to capture operators' nuances

Part of me wonders if there's some grand rearchitecture of this test and the variant consistency tests that lets tests like this be created from a few lambdas, but let's not worry about that now